frontend_tests/node_tests/password.js

```diff
@@ -1,12 +1,25 @@
 add_dependencies({
     zxcvbn: 'node_modules/zxcvbn/dist/zxcvbn.js',
+    i18n: 'i18next',
+});
+
+var i18n = global.i18n;
```

Comment:
It's best to completely stub out i18n.

~~~ js
set_global('i18n', {
    t: function (s) { return 'translated: ' + s; },
});